manager/controlapi/service.go

```diff
@@ -133,6 +133,16 @@ func validateTask(taskSpec api.TaskSpec) error {
 		return grpc.Errorf(codes.InvalidArgument, "ContainerSpec: image reference must be provided")
 	}
 
+	mountMap := make(map[string]bool)
+	for _, mount := range container.Mounts {
+		_, exist := mountMap[mount.Target]
```

Comment:
This is really convoluted. You can usually tell there is a problem if you're using an `else` statement. Let's do this:

```go
if _, exists := mountMap[mount.Target]; exists {
  return grpc.Errorf(codes.InvalidArgument, "ContainerSpec: duplicate mount point: %s", mount.Target)
}

mountMap[mount.Target] = true
```